src/runtime/VMInstance.cpp

```diff
@@ -215,10 +215,7 @@ void vmMarkStartCallback(void* data)
 
         auto& v = self->compiledByteCodeBlocks();
         for (size_t i = 0; i < v.size(); i++) {
-            auto cb = v[i]->m_codeBlock;
-            if (LIKELY(!cb->isAsync() && !cb->isGenerator())) {
-                v[i]->m_codeBlock->setByteCodeBlock(nullptr);
-            }
+            v[i]->m_codeBlock->setByteCodeBlock(nullptr);
```

Comment:
I think that `async` or `generator` functions are highly re-executed in a short term, so It might be efficient to remain ByteCodeBlocks of them.
Is this for memory saving?

Reply:
This is for memory saving
and ExecuitionPauser holds ByteCodeBlock